src/TraceEvent/TraceEvent.cs

```diff
@@ -3079,7 +3079,7 @@ public abstract unsafe class TraceEventDispatcher : TraceEventSource
         /// </summary>
         /// <param name="traceFileName">A path to a trace file.</param>
         /// <returns>A TraceEventDispatcher for the given trace file.</returns>
-        public static TraceEventDispatcher GetDispatcherFromFileName(string traceFileName)
+        public static TraceEventDispatcher GetDispatcherFromFileName(string traceFileName, Etlx.TraceLogOptions options = null)
```

Comment:
Per our conversation earlier, we should not mix TraceLogOptions with TraceEventDispatcher as these are logically two different levels of abstraction.  If we're going to make it possible to plumb this through the generic dispatcher API then I think we will want a different class of options here - perhaps a new "TraceEventDispatcherOptions" class.  Fundamentally that's what this is, and the resulting TraceLog just contains events that were provided by the dispatcher using these options.